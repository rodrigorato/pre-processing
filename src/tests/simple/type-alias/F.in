public class Foo {
	public static void main(String[] args) {
		alias bar = new F() { void f () {}};
		alias F = Foo;
		F f = bar;
		f.f();
	}
}