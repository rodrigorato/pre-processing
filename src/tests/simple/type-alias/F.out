public class Foo {
	public static void main(String[] args) {
		
		
		Foo f = new Foo() { void f () {}};
		f.f();
	}
}